algorithms/sorting/bubble-sort/BubbleSort.java

```diff
@@ -1,10 +1,26 @@
 /**
- * A bubble sort implementation in java
+ * A bubble sort implementation for int arrays in java
  * 
  * @author Harry Willis
  */
  public class BubbleSort {
-    public void bubbleSort(int[] a) {
-        
+    public static void bubbleSort(int[] a) {
+        // sorted flag
+        boolean flag = true;
```

Comment:
Choose a better name for `flag`. It's not apparent what `flag` does.

Maybe something like `isNotSorted` or `swapOccurred`.